pkg/asset/manifests/infrastructure.go

```diff
@@ -203,6 +215,25 @@ func (i *Infrastructure) Generate(ctx context.Context, dependencies asset.Parent
 			}
 			config.Status.PlatformStatus.GCP.ResourceTags = resourceTags
 		}
+
+		for _, service := range installConfig.Config.Platform.GCP.ServiceEndpoints {
+			var serviceName configv1.GCPServiceEndpointName
+			var ok bool
+			if serviceName, ok = gcpServiceEndpointNameMap[service.Name]; !ok {
+				logrus.Debugf("failed to set GCP service endpoint in infrastructure: %s", service.Name)
+				continue
+			}
+			config.Status.PlatformStatus.GCP.ServiceEndpoints = append(config.Status.PlatformStatus.GCP.ServiceEndpoints, configv1.GCPServiceEndpoint{
+				Name: serviceName,
+				URL:  service.URL,
+			})
+
+			sort.Slice(config.Status.PlatformStatus.GCP.ServiceEndpoints, func(i, j int) bool {
+				return config.Status.PlatformStatus.GCP.ServiceEndpoints[i].Name <
+					config.Status.PlatformStatus.GCP.ServiceEndpoints[j].Name
+			})
+		}
```

Comment:
question: I am just curious here about the reasons why the endpoint slice needs sorting?

Reply:
My guess is it just makes it more readable in the manifest when it is output. AWS was doing it and I couldn't see a requirement, but it does make it more readable. It is a list so the order is not guaranteed.

Reply:
Also it makes it easier when dealing with the expected values for testing